### ISSUE TITLE
improvement: turn assertions into error handling and notify the user via a message box

### ISSUE THREAD

**Issue by ivica-k** [ivica-k/terraform-validate-explorer#7]
**Use case:**

At the moment, if there is a mismatch between the `error_count` or `warning_count` reported in the validation file and the number of errors or warnings parsed by the `terraform-validate-parser`, an assertion fails and prints an error message.

This mismatch may be a sign of:

- Wrong parsing by the `terraform-validate-explorer`
- Manual edits to the validation file
- Incorrect counts reported by `terraform validate` (highly unlikely)

In any case, the printer error message won't be seen by the user unless running the tool from a command line.

**Expectations:**

- Users are notified about the mismatch with a message box.
- Text in the message box offers a short explanation of what happened.

**Notes:**

Structure of a validation file:
```
{
  "format_version": "1.0",
  "valid": true,
  "error_count": 12,
  "warning_count": 15,
  "diagnostics": [{}]
}
```
